plugins/graphite.go

```diff
@@ -139,7 +139,11 @@ func (g *Graphite) Run() {
 	go g.flusher()
 
 	for m := range g.Metrics {
-		g.sendToGraphite(buildGraphiteName(m), m.ValueAsString(), m.Ts)
+		g.sendToGraphite(
```

Comment:
Should we simply drop the graphite code since we don't use it anymore? Or, I guess not since this is open source.